astronomer/providers/apache/hive/triggers/hive_partition.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+from typing import Any, AsyncIterator, Dict, Tuple
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.apache.hive.hooks.hive import HiveCliHookAsync
+
+1
+
+
+class HivePartitionTrigger(BaseTrigger):
+    """
+    A trigger that fires and it looks for a partition in the given table
```

Comment:
Suggest to use `A trigger that checks for existence of a partition in a given Hive table`